xbmc/application/ApplicationSettingsHandling.cpp

```diff
@@ -120,6 +120,9 @@ void CApplicationSettingsHandling::OnSettingChanged(const std::shared_ptr<const
   if (m_powerHandling.OnSettingChanged(*setting))
     return;
 
+  if (m_volumeHandling.OnSettingChanged(*setting))
```

Comment:
if you see the lines above you can identify the c&p error somebody (probably me) made a while ago. please fix that while we are at it.

Reply:
Done.